spec/measure_repository_service_test_kit/measure_data_requirements_spec.rb

```diff
@@ -0,0 +1,394 @@
+# frozen_string_literal: true
+
+RSpec.describe MeasureRepositoryServiceTestKit::MeasurePackage do
+  let(:error_outcome) { FHIR::OperationOutcome.new(issue: [{ severity: 'error' }]) }
+  let(:group) { suite.groups[5] }
+  let(:measure_id) { 'measure_id' }
+  let(:session_data_repo) { Inferno::Repositories::SessionData.new }
+  let(:suite) { Inferno::Repositories::TestSuites.new.find('measure_repository_service_test_suite') }
+  let(:test_library_response) do
+    FHIR::Library.new({ type: { coding: [{ code: 'module-definition' }] }, dataRequirement: [{ type: 'Condition' }] })
+  end
+  let(:test_library_wrong_type) do
+    FHIR::Library.new({ type: { coding: [{ code: 'logic-library' }] },
+                        dataRequirement: [{ type: 'Condition' }] })
+  end
+  let(:test_measure) do
+    FHIR::Measure.new
+  end
+  let(:test_session) { repo_create(:test_session, test_suite_id: suite.id) }
+  let(:url) { 'http://example.com/fhir' }
+
+  describe 'Server successfully returns Library 200 and Library on $data-requirements with id' do
+    let(:test) { group.tests.first }
+
+    it 'passes if 200 response returned with Library body' do
+      stub_request(:get, "#{url}/Measure/#{measure_id}")
```

Comment:
Do we need to stub this request to check for successful data requirements? I commented it out and the rspec tests still pass